app/controllers/microposts_controller.rb

```diff
@@ -0,0 +1,74 @@
+class MicropostsController < ApplicationController
+  before_action :set_micropost, only: [:show, :edit, :update, :destroy]
```

Comment:
```suggestion
  before_action :set_micropost, only: %i(show edit update destroy)
```

こういったシンボルの配列に関しては %記法が使えるので、こっちを推奨してます！
https://qiita.com/mogulla3/items/46bb876391be07921743